src/recurrent/matrix/index.ts

```diff
@@ -21,64 +23,83 @@ class Matrix {
    *
    * @param {Number} row
    * @param {Number} col
-   * @returns {Float32Array|Array}
+   * @returns {Number}
```

Comment:
@robertleeplummerjr Can you please check this piece?

Reply:
I believe this is an unused method that was aimed at mirroring https://github.com/karpathy/recurrentjs/blob/84d69349c5412ddebd2b1cbc9b29861ddf3a8169/src/recurrent.js#L63

Since this should actually be "getWeight", non-plural.